src/Storages/StorageFile.cpp

```diff
@@ -377,271 +375,271 @@ Strings StorageFile::getPathsList(const String & table_path, const String & user
 
 namespace
 {
```

Comment:
All the code in this namespace was not changed but the indent was changed. It creates big diff unrelated to the PR and makes it a bit difficult to review